### DISCUSSION TITLE
Blackbox heuristics in EvoMaster

### DISCUSSION THREAD

**Discussion by ahmayun** [EMResearch/EvoMaster#1040] [General]
I wanted to know if EvoMaster implements any default fitness function in blackbox mode, if so what are these doing. According to my understanding, branch distance is one heuristic but is only possible in whitebox mode.

**Comment by arcuri82** [comment]
Hi @ahmayun ,

 Thanks for your interest in EvoMaster. As of version 3.0.0, black-box is just a rather simple random search, with no specific heuristic. Some will be added later this year, as under development.
